### PR TITLE
Thoughts

### PR DESCRIPTION
**Pull request by will-faro**

### CODE REVIEW

File: npm-setup/action.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
-name: 'Set up nmp and node'
-description: 'Set up nmp and node'
+name: 'Set up npm and node'
+description: 'Initialize npm and node versions and authentication'
 
 inputs:
   node-version:
     required: false
-    default: 18
+    default: '18'
```
Comment (will-faro) [inline npm-setup/action.yml:7]: small change to match the spec

File: prerelease-version-monorepo/action.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-name: 'Version and publish'
-description: 'Version and publish'
+name: 'Version and push'
+description: 'Bump package.json version to next pre-release and push commit to git'
```
Comment (will-faro) [inline prerelease-version-monorepo/action.yml:2]: nit: publish is overloaded, this just pushes to git, no NPM publishing involved

File: slack-notify/action.yml [hunk 3]
```diff
@@ -1,17 +1,18 @@
 name: 'Slack notification'
-description: 'Slack notification: notifies on failure and when a failure is fixed'
+description: 'Slack notification: notifies on success->failure and failure->success'
 
 inputs:
+  # TODO include repository/module name in SLACK_MESSAGE
```
Comment (will-faro) [inline slack-notify/action.yml:5]: After a day of seeing these messages, this would be really nice =)

File: docker-build-publish/action.yml [hunk 4]
```diff
@@ -1,16 +1,16 @@
-name: 'Docker build and publish'
-description: 'Docker build and publish'
+name: 'NodeJS Service Docker build and publish'
+description: 'Docker build and publish for Node-based API service containers'
 
 inputs:
   version:
     required: true
-    description: 'A docker version to build'
+    description: 'The image version/tag'
```
Comment (will-faro) [inline docker-build-publish/action.yml:7]: Not required for today, but I wonder if we want the consumer to be able to supply a collection of tags

File: slack-notify/action.yml (hunk 3 above)
Comment (agorina) [reply slack-notify/action.yml]: Done

**Review by agorina — APPROVED** [review]
TYVM

**Pull request merged**